bundle/Resources/public/js/views/cof-contentcreationview.js

```diff
@@ -382,16 +382,52 @@ YUI.add('cof-contentcreationview', function (Y) {
          * @param event {Object} event facade
          */
         _renderSuggestedLocations: function (event) {
-            var suggestedList = this.get('container').one(SELECTOR_SUGGESTED_LOCATIONS),
+            var locations = event.newVal,
+                suggestedList = this.get('container').one(SELECTOR_SUGGESTED_LOCATIONS),
                 itemTemplate = this.get('suggestedItemTemplate'),
                 documentFragment = Y.one(document.createDocumentFragment()),
                 renderedItem;
 
-            event.newVal.forEach(function (location) {
-                var locationPath = '/' + location.get('contentInfo').get('name');
+            locations.forEach(function (location) {
+                var locationPath = location.get('path'),
+                    locationName = location.get('contentInfo').get('name'),
+                    pathSeparator = '/',
+                    longPathSeparator = '.../',
```

Comment:
The separators should be defined outside the forEach loop. In the forEach loop please define only variables that changes their values when loop is running.